testing/bot-integration/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "tsc -b",
     "clean": "rimraf lib tsconfig.tsbuildinfo",
-    "lint": "eslint . --ext .js,.ts",
+    "lint": "eslint .",
```

Comment:
This project still has a .eslintrc.json file

Reply:
fixed